src/scenes/Registration/sections/RaceDemographics/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import styles from './style.module.scss';
+import CheckboxField from 'components/form/CheckboxField';
+
+const raceOptions = [
+  { label: 'American Indian or Alaskan Native', value: 'American Indian or Alaskan Native' },
+  { label: 'Asian', value: 'Asian' },
+  { label: 'Black or African American', value: 'Black or African American' },
+  { label: 'Native Hawaiian or Pacific Islander', value: 'Native Hawaiian or Pacific Islander' },
+  { label: 'Hispanic or Latino or Spanish Origin', value: 'Hispanic or Latino or Spanish Origin' },
+  { label: 'White', value: 'White' },
+  { label: 'Other', value: 'Other', isOther: true },
+  { label: 'I’m optional! :) (Prefer not to answer)', value: 'Prefer Not to Say' },
```

Comment:
This phrasing is weird. What does "I'm optional" mean?

Reply:
:( I thought it was a cute way to ask the race question in a less dramatic way

Reply:
Jk, I think just "Optional! :) (Prefer not to answer)" would be better